website/grading.rst

```diff
@@ -3,92 +3,128 @@
 
-In other words, our late submission policy applies to *ordinary* circumstances: if you are having a busy week, if your RSO has an event that overlaps with a deadline, etc., then you should rely on the resubmission policy to give you some extra flexibility: instead of trying to argue for a few extra days to work on an assignment, you should just submit the work you have completed by the deadline, so you can get feedback on that work and use that feedback to improve your work.
+For the second warm-up exercise (building familiarity with using GitHub collaboratively), teams
+that do not earn an S will be given specific goals to accomplish during Spring #1 of the project
+to demonstrate that they understand how to use GitHub in the context of the project. As long
+as those goals are met, the score will be changed to an S.
+
+Peer Feedback Exercises
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The peer feedback exercises contribute a single SNU score to your final grade.
+
+There are two rounds of peer feedback, where you will be assessed on the quality
+of the feedback you deliver to your peers (you will *not* be assessed based on
+the feedback others give you).
+
+If there are any concerns with the quality of
+your first round of feedback, you will be told exactly what needs to improve in
+the second round of feedback. As long as those issues are addressed,
+you will receive an S in your peer feedback.
+
+Final Project Presentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+During exam week, each team will deliver a final presentation on the work
+they did on the project. Your final presentation will contribute a single ESNU
+score to your final grade (you will be given a concrete rubric for how
+that ESNU score will be determined).
 
 Final Grade
 -----------
 
 Your final grade will be computed based on the following scores:
 
-- 8 ESNU scores from the homeworks.
-- 1 ESNU score from the Project Design Warm-up
+- 7 ESNU scores from the homeworks (of which, at most five will be E's)
+- 2 SNU scores from the Project Warm-up Exercises
 - 1 SNU score from the Peer Feedback
 - 1 ESNU score from the Final Presentation
 - The "Project Points" you accrue in the course project.
 
-In other words, there are 11 ESNU scores (of which, at most 10 can be an E).
+In other words, there are 11 ESNU scores (of which, at most 6 can be an E).
 
 You should avoid thinking of these scores as making up some percent of your grade.
-For example, the fact that there are 8 scores for the homeworks, and 3 scores related
-to the project, does not mean that the homeworks are worth 72% of your grade.
+For example, the fact that there are 7 scores for the homeworks, and 4 scores related
+to the project, does not mean that the homeworks are worth 63% of your grade.
 
 Instead, your grade will be determined based on the table below, where each row
 specifies the minimum number of scores that you must achieve to earn that grade. You
```

Comment:
code block with the CSS class appears in final document:
![image](https://github.com/uchicago-cs/cmsc22000/assets/86740254/2ce18f9d-18b1-4598-a425-72529461b581)